packages/flutter/lib/src/foundation/_isolates_io.dart

```diff
@@ -45,8 +46,13 @@ Future<R> compute<Q, R>(isolates.ComputeCallback<Q, R> callback, Q message, { St
       result.completeError(exception, stack);
     }
   });
+  exitPort.listen((dynamic exitData) {
+    if (!result.isCompleted) {
+      result.completeError(Exception('Isolate exited without result or error.'));
+    }
+  });
   resultPort.listen((dynamic resultData) {
-    assert(resultData == null || resultData is R);
+    assert(resultData is R);
```

Comment:
This change broke a `compute` function for us, which returned null, starting in Flutter 1.20.x. I assume this is resolved by enabling the nnbd feature and returning an optional via compute?

Reply:
Do you have nullsafety enabled on your project?

Reply:
@a14n no.

Reply:
This should be fixed by #63515